elastic/kibana
# [Guided onboarding] Add cloud checks

Partially addresses https://github.com/elastic/kibana/issues/143259

This PR adds cloud checks for the getting started page and guided onboarding plugin.

The cloud checks for the home page and help link will be addressed via https://github.com/elastic/kibana/pull/143837 and https://github.com/elastic/kibana/pull/143765.

### How to test
With cloud enabled (Set `xpack.cloud.id` in your `kibana.dev.yml` file)

1. Verify the "Setup guide" button appears in the header
2. Verify you can navigate to `/app/home#/getting_started`

With cloud disabled (Remove `xpack.cloud.id` in your `kibana.dev.yml` file)

1. Verify the "Setup guide" button **does not** appear in the header
2. Verify navigating to `/app/home#/getting_started` redirects back to the Kibana home page.
